documentation/src/main/asciidoc/getting-started.asciidoc

```diff
@@ -245,10 +245,19 @@ include::{sourcedir}/org/hibernate/search/documentation/gettingstarted/withhsear
 
 However, keep in mind that data already present in your database when you add the Hibernate Search integration
 is unknown to Hibernate Search, and thus has to be indexed through a batch process.
+Use the mass indexer api to do that, as shown in the following code:
```

Comment:
Maybe "To that end, you can use the mass indexer API, as shown [...]"? There are (or at least will be) other options.